ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -632,6 +632,30 @@ Loader {
                         return "";
                     }
 
+                    album: {
+                        if (contentType !== StatusMessage.ContentType.Image) {
+                            return []
+                        }
+                        let message = root.messageStore.getMessageByIdAsJson(responseToMessageWithId)
```

Comment:
Using `getMessageByIdAsJson` is slow, so we want to limit it as much as possible.

The best way would be to add `albumMessageImages` and `albumImagesCount` to the message model directly as `quotedAlbumMessageImages` and `quotedAlbumImagesCount`.

If it's not possible to add it to the model, then we could move `let message = root.messageStore.getMessageByIdAsJson(responseToMessageWithId)` as a component property above (using conditions on image and sticker) so that we only call that function once.